test/test.ts

```diff
@@ -0,0 +1,96 @@
+import * as fs from 'fs';
+import * as test from 'tape';
+import { ReferenceTranscoder } from './reference';
+
+const NUM_MODES = 19;
+const SIZE: number = process.env.CI ? 512 : 256;
+
+const nBlocks = 8 * SIZE * SIZE;
+const compressedByteLength = nBlocks * 16;
+const uncompressedByteLength = SIZE * ((SIZE + 3) >> 2) * 4 * 4;
```

Comment:
The docs for RGBA32 show two lines that look contradictory:

```js
const uncompressedByteLength = width * yBlocks * 4 * 4;
```

```js
const textureByteLength = width * height * 4;
let decodedTextureView = new Uint8Array(memory.buffer, 65536 + compressedByteLength, textureByteLength);
```

Shouldn't `uncompressedByteLength` and `textureByteLength` match?

Reply:
These are correct. When the texture's height is a multiple of 4, these lines evaluate to the same value. In other cases, `uncompressedByteLength` would be greater than `textureByteLength` because the decoder uses that memory anyway.